tests/test_quantum_info_entanglement.py

```diff
@@ -136,11 +136,11 @@ def test_meyer_wallach_entanglement(backend):
     circuit2.add(gates.CNOT(0, 1))
 
     backend.assert_allclose(
-        meyer_wallach_entanglement(circuit1, backend=backend), 0.0, atol=PRECISION_TOL
+        meyer_wallach_entanglement(circuit1, backend=backend), 0.0, atol=1e-6, rtol=1e-6
```

Comment:
why do we need to change the tolerance (reduce?) in a bunch of tests here and below? Is it always related to `pytorch` using `float32`?

Reply:
Now I should have set torch for using float64 by default. However there are still many cases when it is necessary to set tolerance to 10-6 (usually errors of the order of 10-7 appears). I think this may be related to a slightly different ways to implement the operations.